### ISSUE TITLE
how to employ sampled_softmax_loss in bert?

### ISSUE THREAD

**Issue by monologue1107** [tensorflow/models#10017]
I am using official/nlp/bert to achieve classifier tasks. And I need to use sampled_softmax_loss function instead of the original softmax one. The definition of sampled_softmax_loss function are showed below:

tf.nn.sampled_softmax_loss(weights, # Shape (num_classes, dim)     - floatXX
                     biases,        # Shape (num_classes)          - floatXX 
                     labels,        # Shape (batch_size, num_true) - int64
                     inputs,        # Shape (batch_size, dim)      - floatXX  
                     num_sampled,   
                     num_classes,   
                     num_true=1,    
                     sampled_values=None,
                     remove_accidental_hits=True,
                     partition_strategy="mod",
                     name="sampled_softmax_loss")

However, I cannot find where _weights_ and _biases_ are in BERT model. (I can only get labels and inputs.) Can anyone help me with it?

**Comment by laxmareddyp** [comment]
Hi @monologue1107,

Thank you for opening this issue. Since this issue has been open for a long time, the code/debug information for this issue may not be relevant with the current state of the code base.
The TF models official team is constantly improving the framework by fixing bugs and adding new features. We suggest you try the latest TensorFlow [version](https://pypi.org/project/tf-models-official/) with the latest compatible hardware configuration which could potentially resolve the issue. If you are still facing the issue, please create a new GitHub issue with your latest findings, with all the debugging information which could help us investigate.
Please follow the [release notes](https://github.com/tensorflow/models/releases/tag/v2.13.1) to stay up to date with the latest developments which are happening in the TF models official space.
